scripts/take_photo.py

```diff
@@ -1,3 +1,43 @@
-import pi_camera
+#!/usr/bin/env python3
 
-pi_camera.take_photo([0, 0])
+import argparse
+import time
+
+import numpy as np
+
+import context
+import roboplot.config as config
+import roboplot.core.curves as curves
+import roboplot.core.hardware as hardware
+from roboplot.core.gpio.gpio_wrapper import GPIO
+from roboplot.core.camera.camera_wrapper import Camera
+
+try:
+    # Commandline arguments
+    parser = argparse.ArgumentParser(description='Draw a circle.')
```

Comment:
Changed below will wait to merge until you have tested it on the pi. Please can you commit any tweaks you make to get it working on the pi.